docs/running-in-production/admin-user.md

```diff
@@ -11,19 +11,29 @@ psql postgres
 - Check the users table by running
 
 ```sql
-SELECT * FROM users;
+SELECT
+  *
+FROM
+  users;
 ```
 
 Which will display a table of users in the database:
 
 ```sql
```

Comment:
I'd revert the below change and make this code block `txt` instead?

Reply:
Done in b2cbf6a32.